### DISCUSSION TITLE
Updated sostat for SecOnion 2.3

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#7998] [Show and tell] [answered]
I took the sostat from 16.04 and tried to make it useable for 2.3.  We have a lot of remote users who require support, and the SOC is great but getting the dashboards from say Granfana sent to us requires us to still collect data like the sostat.  

It's crude and needs work, but I removed a lot of the 16.04 references and tried to make it useful.  I am also sure there are more things to add to this, and probably better ways. 
Sorry tried to upload a txt would not accept, so made it a docx file.

https://gist.github.com/xfaith/a3a4af08d1c9e83be0d59c58888f4cd1

**Comment by isaacgolding** [comment]
I just tried this out and the only error I noticed was as follows: 

================================================
Error: No such container: so-idstool

**Reply by xfaith** [reply]
Got it, I noticed it also, its supposed to be so-idstools.  I have made some changes/updates since I last posted it.

**Comment by defensivedepth** [comment, marked as answer]
@xfaith I have removed the attached word doc.

Per https://github.com/Security-Onion-Solutions/securityonion/discussions/1720, for security reasons, please avoid uploading any non-plain text files. 

You could post it on https://gist.github.com/ and link that here as well.

**Reply by xfaith** [reply]
Got it , sorry didnt see that part.

**Reply by xfaith** [reply]
https://gist.github.com/xfaith/a3a4af08d1c9e83be0d59c58888f4cd1
Lets see if that works.  Needs work.  Wanted to try and use the so-status and add Mem Usage/CPU usage, but right now it just spits out both.   There are other things of course that could do with some updates.

**Reply by defensivedepth** [reply]
Yep that gist is better than a docx, thanks!
